package.json

```diff
@@ -1,40 +1,77 @@
 {
-    "name": "hyperdrive",
-    "version": "1.0.0",
-    "description": "Hyperdrive Smart Contracts",
-    "engines": {
-        "node": "^18"
-    },
-    "scripts": {
-        "postinstall": "patch-package --patch-dir .patches",
-        "configure-husky": "npx husky install && npx husky add .husky/pre-commit \"npx --no-install lint-staged\""
-    },
-    "_moduleAliases": {
-        "test": "./test"
-    },
-    "lint-staged": {
-        "**/**/*.sol": [
-            "npx prettier --write"
-        ]
-    },
-    "husky": {
-        "hooks": {
-            "pre-commit": "lint-staged"
-        }
-    },
-    "license": "Apache-2.0",
-    "devDependencies": {
-        "coveralls": "^3.1.1",
-        "cspell": "^6.1.1",
-        "husky": "^8.0.0",
-        "lint-staged": "^13.0.0",
-        "patch-package": "^6.4.7",
-        "prettier": "^2.6.2",
-        "prettier-plugin-solidity": "^1.0.0-beta.19",
-        "solhint": "^4.5.2",
-        "solhint-plugin-prettier": "0.0.5"
-    },
-    "dependencies": {
-        "yarn": "^1.22.19"
+  "name": "hyperdrive",
```

Comment:
Same here. Looks like we're reformatting a lot of files in this PR. As said earlier, I'm not a fan of two spaces, but it's not a hill I'm going to die on if there is a good reason for making the change.